var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -52,35 +65,25 @@ def _ld_library_path(self):
             yield self._join_prefix(dir)
 
     def install(self, spec, prefix):
-        # For quick turnaround debugging, comment out line below and
-        # use the copy instead
+        # install cpp
+        # Copy instead of install to speed up debugging
+        # subprocess.run(f'cp -r /opt/intel/oneapi/compiler {prefix}', shell=True)
         super(IntelOneapiCompilers, self).install(spec, prefix)
-        # Copy installed compiler instead of running the installer
-        # from shutil import copytree
-        # copytree('/opt/intel/oneapi/compiler', join_path(prefix, 'compiler'),
-        #         symlinks=True)
 
+        # install fortran in same prefix
+        self.install_from_releases(spec, prefix, 'fortran-installer',
+                                   'fortran-compiler', fortran_releases)
+
+        # set rpath so 'spack compiler add' can check version strings
+        # without setting LD_LIBRARY_PATH
         rpath = ':'.join(self._ld_library_path())
         patch_dirs = ['compiler/lib/intel64_lin',
                       'compiler/lib/intel64',
                       'bin']
         for pd in patch_dirs:
-            for file in glob.glob(self._join_prefix(join_path(pd, '*'))):
+            patchables = glob.glob(self._join_prefix(path.join(pd, '*')))
```

Comment:
I don't see `path` defined in this function (i.e. I think there may be a problem with `path.join(pd, '*')`)

Reply:
At the top of the file, I do:

`from os import path`

I will change if there is a preferred idiom